src/main/java/Duke.java

```diff
@@ -1,10 +1,125 @@
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke\nWhat can I do for you?");
+        Scanner in = new Scanner(System.in);
+        boolean isComplete = false;
+
+        while (!isComplete) {
+
+            String line = in.nextLine();
+            String firstWord = getFirstWord(line);
+            Task[] taskList = Task.getTaskList();
+
+            switch (firstWord) {
+            case "mark":
+                markTask(taskList, Integer.parseInt(getSecondWord(line)));
+                break;
+            case "unmark":
+                unmarkTask(taskList, Integer.parseInt(getSecondWord(line)));
+                break;
+            case "list":
+                listTask(taskList);
+                break;
+            case "todo":
+                getTodoDetails(line);
+                break;
+            case "deadline":
+                getDealineDetails(line);
+                break;
+            case "event":
+                getEventDetails(line);
+                break;
+            case "bye":
+                isComplete = true;
+                break;
+            default:
+                System.out.println("Unrecognized command. Please try again.");
+                break;
+            }
+        }
+
+        in.close();
+        System.out.println("Bye. Hope to see you again soon!");
+    }
+
+    private static void getTodoDetails(String line) {
+        String todoLine = getSecondWord(line);
+        Todo todoTask = new Todo(todoLine);
+        Task.addTask(todoTask);
+        Task.upTaskCount();
+        System.out.println("Got it. I've added this task:\n" + todoTask.describeTask());
+        System.out.println("\nNow you have " + Task.getTaskCount() + " tasks in the list.\n");
+    }
+
+    private static void getDealineDetails(String line) {
+        String deadlineLine = getSecondWord(line);
+        int index = deadlineLine.indexOf("/");
+        String deadlineDescription = deadlineLine.substring(0, index - 1);
+        String deadlineDate = deadlineLine.substring(index + 4);
+        Deadline deadlineTask = new Deadline(deadlineDescription, deadlineDate);
+        Task.addTask(deadlineTask);
+        Task.upTaskCount();
+        System.out.println("Got it. I've added this task:\n" + deadlineTask.describeTask());
+        System.out.println("\nNow you have " + Task.getTaskCount() + " tasks in the list.\n");
+    }
+
+    private static void getEventDetails(String line) {
+        String eventLine = getSecondWord(line);
+        int index2 = eventLine.indexOf("/");
+        String eventDescription = eventLine.substring(0, index2 - 1);
+        String eventDate = eventLine.substring(index2 + 4);
+        int index3 = eventDate.indexOf("/");
+        String eventFrom = eventDate.substring(2, index3 - 1);
+        String eventTo = eventDate.substring(index3 + 4);
```

Comment:
Maybe can consider abstracting this part out into a separate method so that the whole getEventDetails method will have the same level of abstraction